### ISSUE TITLE
Accessibility: Lack of `aria-*` and `role` attributes 

### ISSUE THREAD

**Issue by cvrebert** [Dogfalo/materialize#850]
Other than the pickadate.js-powered datepicker, it doesn't look like any of the JavaScript widgets make use of the `role` attribute or `aria-*` attributes needed for proper web accessibility support.
Here's an overview: http://a11yproject.com/posts/getting-started-aria/

**Comment by Merovex** [comment]
See rant in #1424

**Comment by Dogfalo** [comment]
Duplicate of #1424
